packages/charting/webpack.serve.config.js

```diff
@@ -2,8 +2,6 @@ const path = require('path');
 const resources = require('../../scripts/webpack/webpack-resources');
 
 module.exports = resources.createServeConfig({
-  entry: './src/demo/index.tsx',
```

Comment:
I think this part and the change in webpack-resources were because I tried to also move legacy demo apps (except fabric-website-resources probably) to the examples package, and ran into assorted problems (don't remember the details).

If it proves to be too much of a pain, we might want to just get rid of the legacy demo apps (except fabric-website-resources and the demo app for react-button if we still need that to test polyfills).